types/tsconfig.json

```diff
@@ -6,7 +6,6 @@
     "moduleResolution": "node",
     "emitDeclarationOnly": false,
     "declaration": true,
-    "baseUrl": "./src",
```

Comment:
What's the purpose of that removal? Just to make sure I understand.

Reply:
As per IRL it forces the relative imnport

Reply:
To force use to use relative imports so we cannot have the issue again 👍  Worth digging if we could use `@something` alias